### DISCUSSION TITLE
Frozen Column Header is broken when using with column group

### DISCUSSION THREAD

**Discussion by tmair2-go** [primefaces/community#215] [PrimeVue]
[StackBlitz Link](https://stackblitz.com/edit/vitejs-vite-ezmpy8?file=src%2FApp.vue)

When using column groups and frozen together, the header does not get properly fixed. When using just columns and frozen, there is a slight twisting effect, but it's not a significant issue.

When looking at the elements in the developer tools, I often encounter issues where the left position is calculated without properly accounting for the width of the before columns. This issue seems to occur primarily when there are a large number of headers or when column data lengths cause column width to resize.

Even when not using column groups, there are occasional slight distortions in the columns as I scroll horizontally.

**Comment by tugcekucukoglu** [comment]
Move to the issue https://github.com/primefaces/primevue/issues/4677
